docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -400,177 +575,198 @@ The following features are not supported:
+|------------------|---------------------|-----------------------|------------------------------------------|
+| `ObjectURL`      | `string`            | Yes                   | The location of the `.obj` file.         |
+| `MaterialtURL`   | `string`            | No                    | The location of the `.mtl` file. Leave blank for a textureless object. |
+| `TexturesPath`   | `string`            | Yes if using materials| The path to the directory where the textures (including gloss and normal) are stored. |
+| `ShadingMode`    | `MeshVisualShadingModeValue` | No           | The type of the shading mode that the mesh uses (see tab. [with possible values](#table_mesh_possible_values)). |
+| `UseMipmapping`  | `bool`              | No                    | Specifies whether to use mipmaps for textures (by default `true`). |
+| `UseSoftNormals` | `bool`              | No                    | Specifies whether to average normals at each point to smoother the textures (by default is `true`). |
+| `LightPosition`  | `Vector3`           | No                    | The position, in the stage space, of the point light that applies lighting to the model (all zeros indicate the top-left corner). |
+
+-   The `ShadingMode` property defines the shading mode type used by the mesh.
 
--   You can select from seven `Shape` values, some of which are specializations as compared to another.
+<a name="table_mesh_possible_values"></a>
+**Table: Shading mode values**
 
-    **Table: Shape values**
+| Enumeration value                      | Description                              |
+|----------------------------------------|------------------------------------------|
+| `TexturelessWithDiffuseLighting`       | Simplest, one color that is lit by ambient and diffuse lighting. |
+| `TexturedWithSpecularLighting`         | Uses only the visual image textures provided with specular lighting in addition to ambient and diffuse lighting. |
+| `TexturedWithDetailedSpecularLighting` | Uses all textures provided including a gloss, normal, and texture map along with specular, ambient, and diffuse lighting. |
 
-    | Enumeration value | Description                              |
-    |-----------------|----------------------------------------|
-    | `Sphere`          | The default shape.                                 |
-    | `ConicalFrustrum` | The area bound between two circles (basically, a cone with the tip removed). |
-    | `Cone`            | Equivalent to a conical frustum with a top radius of 0. |
-    | `Cylinder`        | Equivalent to a conical frustum with equal radii for the top and bottom circles. |
-    | `Cube`            | Equivalent to a beveled cube with a bevel percentage of 0. |
-    | `Octahedron`      | Equivalent to a beveled cube with a bevel percentage of 1. |
-    | `BevelledCube`    | A cube/cuboid with all edges flattened to some degree. |
 
 **Usage:**
```

Comment:
suggest the following heading here:
#### Use MeshVisual

What is your opinion?